glam/api/migrations/0025_non_normalized_aggregations.py

```diff
@@ -0,0 +1,81 @@
+# Generated by Django 3.1.13 on 2023-05-25 22:59
+
+from django.db import migrations, models
+
+from glam.api import constants
+
+sql_create = []
+sql_rename = []
+sql_drop = []
+for channel in constants.CHANNEL_NAMES.values():
+    sql_create.extend(
+        [
+            f"CREATE MATERIALIZED VIEW view_glam_desktop_{channel}_aggregation_2 AS SELECT * FROM glam_desktop_{channel}_aggregation;",
+            f"CREATE UNIQUE INDEX ON view_glam_desktop_{channel}_aggregation_2 (id);",
+            f"CREATE INDEX ON view_glam_desktop_{channel}_aggregation_2 (version);",
+            f"CREATE INDEX ON view_glam_desktop_{channel}_aggregation_2 USING HASH (metric);",
+            f"CREATE INDEX ON view_glam_desktop_{channel}_aggregation_2 (os);",
```

Comment:
can we do `v2` instead of just 2, so it matches the naming pattern in ETL too.

Reply:
the _2 here is not related to the _v2 in the ETL table name. This is just a temporary name. I could use `_temp` for it  but it could be interpreted as a temporary Materialized View instead of a temporary name